profiling/src/bindings/mod.rs

```diff
@@ -10,8 +10,13 @@ use std::sync::atomic::AtomicBool;
 
 pub type VmInterruptFn = unsafe extern "C" fn(execute_data: *mut zend_execute_data);
 
+#[cfg(feature = "allocation_profiling")]
 pub type VmMmCustomAllocFn = unsafe extern "C" fn(u64) -> *mut libc::c_void;
+
+#[cfg(feature = "allocation_profiling")]
 pub type VmMmCustomReallocFn = unsafe extern "C" fn(*mut libc::c_void, u64) -> *mut libc::c_void;
+
+#[cfg(feature = "allocation_profiling")]
 pub type VmMmCustomFreeFn = unsafe extern "C" fn(*mut libc::c_void);
```

Comment:
IIRC, you can do this:

```suggestion
#[cfg(feature = "allocation_profiling")]
{
    pub type VmMmCustomAllocFn = unsafe extern "C" fn(u64) -> *mut libc::c_void;
    pub type VmMmCustomReallocFn = unsafe extern "C" fn(*mut libc::c_void, u64) -> *mut libc::c_void;
    pub type VmMmCustomFreeFn = unsafe extern "C" fn(*mut libc::c_void);
}
```

Do you think that's better?

Reply:
Tried it and I'd love to, this would nicely group together what belongs together. Same in [profiling/src/lib.rs](https://github.com/DataDog/dd-trace-php/blob/955d8e10ecb7d2af26863f93c2cf26ae5e0ba5c3/profiling/src/lib.rs#L133-L143), but Rust won't allow it though:
```
error: expected item after attributes
  --> src/bindings/mod.rs:13:1
   |
13 | #[cfg(feature = "allocation_profiling")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Same when trying to make this an inner attribute of a block, Rust expects an item before the opening curly braces (at least at the root level of a file)